### PR TITLE
Landing page style refinement

### PR DESCRIPTION
**Pull request by waidelacruz**
https://trello.com/c/ax1LuaOQ/21-update-landing-pages-on-cascade-to-reflect-style-changes-made-on-the-standalone-pages

### CODE REVIEW

File: app/assets/stylesheets/widgets/_ad_landing_page/masthead.scss [hunk 1]
```diff
@@ -261,11 +269,11 @@
       font-size: 20px;
       line-height: 28px;
       font-weight: 200;
-
+      
       @include media($medium-screen) {
-        @include span-columns(12);
+        /*@include span-columns(12);*/
```
Comment (tatwell) [inline app/assets/stylesheets/widgets/_ad_landing_page/masthead.scss]: This line can be removed.

**Comment by tatwell** [comment]
👍  Looks good.

File: app/assets/stylesheets/widgets/_ad_landing_page/masthead.scss (hunk 1 above)
Comment (waidelacruz) [inline app/assets/stylesheets/widgets/_ad_landing_page/masthead.scss]: I kept that commented out in case we need to revert.

**Comment by mirabelrice** [comment]
👍

File: app/views/widgets/ad_landing_page/_masthead.html [hunk 2]
```diff
@@ -25,14 +25,7 @@ <h2 class="subtitle">Earn your Masters Degree at Dodge College of Film and Media
...
       <div class="calls-to-action">
-        <div id="form_a4517720-e7d7-4e0e-bafd-2c1045918e0d">Loading...</div>
-        <script>/*<![CDATA[*/
-          var script = document.createElement('script');
-          script.async = 1;
-          script.src = 'https://go.chapman.edu/register/?id=a4517720-e7d7-4e0e-bafd-2c1045918e0d&output=embed&div=form_a4517720-e7d7-4e0e-bafd-2c1045918e0d' + ((location.search.length > 1) ? '&' + location.search.substring(1) : '');
-          var s = document.getElementsByTagName('script')[0];
-          s.parentNode.insertBefore(script, s);
-        /*]]>*/</script>
+        <div id="form_4c37abdc-d5ba-4313-99b6-89b7a869ac37">Loading...</div><script async="async" src="https://go.chapman.edu/register/?id=4c37abdc-d5ba-4313-99b6-89b7a869ac37&amp;output=embed&amp;div=form_4c37abdc-d5ba-4313-99b6-89b7a869ac37">/**/</script>
```
Comment (tatwell) [inline app/views/widgets/ad_landing_page/_masthead.html]: Why was this changed? Will this behave the same?

Comment (waidelacruz) [inline app/views/widgets/ad_landing_page/_masthead.html]: This updated code was provided by Shawn and will eventually change depending on the campaign.
